### DISCUSSION TITLE
Give tips on how to optimize the use of Gotenberg.

### DISCUSSION THREAD

**Discussion by starwalkn** [gotenberg/gotenberg#1064] [Q&A] [answered]
I have one service that receives events to generate documents, which it sends to a single Gotenberg instance. The service itself also exists in a single instance. The first 5-6 requests per second load the system, which has about 2000 Mib CPU and 2 GB RAM. Is there any way to optimize Gotenberg itself? For example, I can select only the HTML to PDF conversion module (chromium) and disable all other functions. We are using Gotenberg 7.8.

**Comment by gulien** [comment, marked as answer]
No real way to optimize Gotenberg. Maybe at the orchestrator level?

**Reply by starwalkn** [reply]
Looks like yes, we'll have to do it at the orchestrator level. Thanks for the reply!
